src/utils/email.js

```diff
@@ -133,7 +133,7 @@ function getUnsubscribeCtaHref ({ subscriber, isMonthlyEmail }) {
   }
 
   if (isMonthlyEmail && !subscriber.primary_verification_token) {
-    throw new Error('subscriber has no primary verification_token')
+    throw new MethodNotAllowedError('subscriber has no primary verification_token')
```

Comment:
Err 405 "indicates that the web server has recognized a request from a web browser to access the website but rejects the specific HTTP method"

This seems like a 400 or 401